frontend/packages/console-app/src/components/nav/FavoriteNavItemResources.scss

```diff
@@ -0,0 +1,38 @@
+.oc-favorite-resource.pf-v6-c-nav__item {
+  .pf-v6-c-nav__link {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    flex-grow: 1;
+    overflow: hidden;
+    position: relative;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    padding-top: 0;
+    padding-bottom: 0;
+    padding-right: 0;
+    &:hover {
+      --pf-v6-c-nav__section-title--PaddingRight: 30px;
+      .oc-favorite-delete-button {
+        .pf-v6-c-button__text {
+          opacity: 1;
+        }
+      }
+    }
+  }
+}
+
+.oc-favorite-delete-button {
+  background-color: transparent !important;
+  margin-left: auto;
+  margin-right: var(--pf-t--global--spacer--lg);
+}
+
+.oc-favorite-menu {
```

Comment:
Use PatternFly spacing utility instead.
```suggestion
```

Reply:
If I remove this styling, there is padding-top and padding-bottom added extra. 

<img width="1724" alt="image" src="https://github.com/user-attachments/assets/12aa0b73-a8ed-4913-bf9c-86719d6ee6ba" />